examples/model_selection/plot_precision_recall.py

```diff
@@ -69,7 +69,8 @@
 
 where :math:`P_n` and :math:`R_n` are the precision and recall at the
 nth threshold. A pair :math:`(R_k, P_k)` is referred to as an
-*operating point*.
+*operating point*. When summarizing a precision-recall curve, AP is preferable
+to computing the trapezoidal area under the operating points.
```

Comment:
My main gripe is the corner case where a classifier predicts only a single score, or scores that are close enough to be treated as equivalent.  Computing the area with the trapezoid rule in that case gives area > 0.5 to a poor classifier.  I've seen it cause problems for students trying to do automated hyperparameter searches based on AUCPR.

But whatever is going to be the reference AUCPR value should be recommended here.  If that's not AP, I'll change it.